README.md

```diff
@@ -119,3 +119,17 @@ Options:
                                 unavailable
                                          [default: "https://registry.npmjs.com"]
 ```
+
+## Configuration File
+
+We use [`rc`](https://github.com/dominictarr/rc) to parse configuration files. Please see the [`rc` repository](https://github.com/dominictarr/rc#standards) for the order of precedence used when searching for configuration files. Our app is `ipfs-npm`.
+
+Please see the rc repository for the order of precedence used when searching for configuration files.
```

Comment:
This sentence is repeated.  Please remove.

Reply:
@achingbrain Ugh. Sorry. Done!